src/explore-education-statistics-admin/src/pages/bau/FeedbackPage.tsx

```diff
@@ -0,0 +1,137 @@
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import React from 'react';
+import FormattedDate from '@common/components/FormattedDate';
+import { useQuery } from '@tanstack/react-query';
+import feedbackQueries from '@admin/queries/feedbackQueries';
+import feedbackService from '@admin/services/feedbackService';
+import ButtonText from '@common/components/ButtonText';
+import InsetText from '@common/components/InsetText';
+import ButtonLink from '@admin/components/ButtonLink';
+import useQueryParams from '@admin/hooks/useQueryParams';
+import { FeedbackViewModel } from '@common/services/types/feedback';
+import truncateAround from '@common/utils/string/truncateAround';
+import FeedbackDetailsModal from '../admin-dashboard/components/FeedbackDetailsModal';
+
+type Params = {
+  showRead?: string;
+};
+
+const FeedbackPage = () => {
+  const {
+    data: feedbackItems = [],
+    isLoading: isLoadingFeedback,
+    refetch: reloadFeedbackItems,
+  } = useQuery(feedbackQueries.getFeedback);
+
+  const toggleReadStatus = async (id: string) => {
+    await feedbackService.toggleReadStatus(id);
+    reloadFeedbackItems();
+  };
+
+  const { showRead } = useQueryParams<Params>();
+
+  const getResponseText = (response: FeedbackViewModel['response']) => {
+    switch (response) {
+      case 'Useful':
+        return 'Useful';
+      case 'NotUseful':
+        return 'Not useful';
+      case 'ProblemEncountered':
+        return 'Problem encountered';
+      default:
+        return '';
+    }
+  };
```

Comment:
This can be hoisted to module-level as it's a pure function with only inputs and outputs. 

We can also export this so it can just be used in the `FeedbackDetailsModal` directly. This way we don't need to pass this function into the component as a prop and can just import it into the component.